doc/subsystems/networking/usbnet_setup.rst

```diff
@@ -0,0 +1,79 @@
+.. _usb_device_networking_setup:
+
+USB Device Networking
+#####################
+
+This page describes how to set up networking between a Linux host
+and a Zephyr application running on USB supported devices. A sample
+application from Zephyr source distribution is run on one of
+supported boards. The board is connected to Linux host using USB cable
+and provides an Ethernet interface to the host.
+
+Basic Setup
+***********
+
+To communicate with Zephyr application over newly created Ethernet
+interface we need to assign IP addresses and set routing table.
+
+After plugging USB cable to Linux host cdc_ether driver registers a new
+Ethernet device with provided MAC
+
+.. code-block:: console
+
+   cdc_ether 1-2.7:1.0 eth0: register 'cdc_ether' at usb-0000:00:01.2-2.7, CDC Ethernet Device, 00:00:5e:00:53:01
```

Comment:
Where is this being displayed?  Do you use dmesg on the Linux host to see this?

Reply:
yes, dmesg also